cumulusci/core/metadeploy/tests/test_api.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+import responses
+from requests.models import Response
+from requests.sessions import Session
+from responses import matchers
+
+from cumulusci.core.config import ServiceConfig
+from cumulusci.core.exceptions import CumulusCIException
+from cumulusci.core.metadeploy.api import MetaDeployApi, make_api_session
+
+DEFAULT_REST_URL: str = "https://metadeploy.example.com/api/rest"
+DEFAULT_TOKEN: str = "37b003aee9bdd744a6618e9fe12"
+
+
+@pytest.fixture
+def default_api_client():
+    return MetaDeployApi(
+        metadeploy_service=ServiceConfig(
+            {"url": DEFAULT_REST_URL, "token": DEFAULT_TOKEN}
+        )
+    )
+
+
+@pytest.fixture
+def auth_header_matcher():
+    return matchers.header_matcher({"Authorization": f"token {DEFAULT_TOKEN}"})
+
+
+@responses.activate
+def test_make_api(auth_header_matcher):
+    expected_result: dict = {"data": [1]}
+    responses.add(
+        "GET",
+        "https://metadeploy/api/rest",
+        json=expected_result,
+        match=[auth_header_matcher],
+    )
+    responses.add(
+        "POST",
+        "https://metadeploy/api/rest",
+        json=expected_result,
+        match=[auth_header_matcher],
+    )
+    service: ServiceConfig = ServiceConfig(
+        {"url": "https://metadeploy/api", "token": DEFAULT_TOKEN}
+    )
+    api_session: Session = make_api_session(service)
+    resp: Response = api_session.get("/rest")
```

Comment:
So if I'm reading this correctly you can use either the absolute _or_ relative url with session passed back from `make_api_session()`?